native-base-theme/variables/material.js

```diff
@@ -256,13 +256,13 @@ export default {
   noteFontSize: 14,
 
   // Title
-  titleFontfamily: platform === "ios" ? "Titillium Web" : "Titillium Web_Regular",
-  titleFontSize: 19,
+  titleFontfamily: platform === "ios" ? "Titillium Web" : "Titillium Web",
```

Comment:
I wonder, since we have most styled defined here, why we need to repeat these styles in every component? @fpersico